subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/MavenConversionIntegrationTest.groovy

```diff
@@ -139,7 +140,7 @@ Root project 'webinar-parent'
     def "flatmultimodule"() {
         def dsl = dslFixtureFor(scriptDsl as BuildInitDsl)
         executer.beforeExecute {
-            executer.inDirectory(targetDir.file("webinar-parent"))
+            executer.inDirectory(targetDir.file("webinar-parent")).withWarningMode(WarningMode.None) // FIXME we cannot assert warnings in this test as withWarningMode is ignored for the Kotlin DSL
```

Comment:
This is interesting. Is this something caused by our test fixtures?

Reply:
Probably. I've copy-pasted the sample project to a local folder and executed the tests manually and it all worked as expected. I've tried to debug the test fixture, but it simply didn't work. I wasted too much time on this, so I ended up just adding a `fixme` message. I thought this test will go away in 8.0 anyway.